package.json

```diff
@@ -100,7 +100,7 @@
     "chalk": "^2.3.0",
     "common-tags": "^1.7.2",
     "dag-map": "^2.0.2",
-    "ember-cli": "2.18.0",
+    "ember-cli": "github:ember-cli/ember-cli#6a85734c52d129fc0824ec3fbf21bd01f9c9fa38",
```

Comment:
I can't point to `` as intended as experiments are removed from betas and releases. Pointing to https://github.com/ember-cli/ember-cli/commit/6a85734c52d129fc0824ec3fbf21bd01f9c9fa38 instead